.github/workflows/build-and-test.yml

```diff
@@ -28,6 +28,13 @@ jobs:
       with:
         toolchain: stable
         override: true
+      
+    - name: Remove msys64 # Workaround to resolve link error with C:\msys64\mingw64\bin\libclang.dll
+      if: runner.os == 'Windows'
+      run: Remove-Item -LiteralPath "C:\msys64\" -Force -Recurse
+    - name: Install dependencies
+      if: runner.os == 'Windows'
+      run: choco install llvm -y
```

Comment:
It might be worth using [KyleMayes/install-llvm-action](https://github.com/KyleMayes/install-llvm-action) here (like the Java binding does). For some reason it's much faster than using `choco`

Reply:
Addressed in #489